jsk_2016_01_baxter_apc/euslisp/jsk_2016_01_baxter_apc/baxter-interface.l

```diff
@@ -394,20 +395,19 @@
         )
       (when do-stop-grasp (unless graspingp (send self :stop-grasp arm)))
       (send self :angle-vector-sequence
-            (list
-              (send *baxter* arm :move-end-pos #f(0 0 50) :world)
-              (send self :ik->bin-entrance arm bin)
-              )
-             :fast nil 0 :scale 5.0)
+         (reverse avs)
+         ;;(list
```

Comment:
please write comment why you did not remove this line, keep code as commented-out